openmc/material.py

```diff
@@ -41,11 +41,19 @@ class Material(object):
     name : str, optional
         Name of the material. If not specified, the name will be the empty
         string.
+    temperature : str, optional
+        The temperature identifier applied to this material. The units are
+        in Kelvin and the temperature rounded to the nearest integer.
+        For example, a tempreature of 293.6K would be provided as '294K'
 
     Attributes
     ----------
     id : int
         Unique identifier for the material
+    temperature : str
```

Comment:
We actually already have temperature as a property of `Cell`, so it seems redundant to me to have temperature specified on the material. How easy would it be to revert to using the cell temperature?